static/src/javascripts/projects/common/modules/identity/api.js

```diff
@@ -206,3 +214,16 @@ export const updateUsername = (username: string): any => {
 
     return request;
 };
+
+export const ajaxSignIn = (credentials: PasswordCredential) => {
+    const url = `${profileRoot || ''}/actions/auth/ajax`;
+    return fetch(url, {
```

Comment:
this is for @guardian/guardian-frontend-team to decide but I think we need to use `reqwest` instead of real `fetch`

Reply:
as far as I can tell, this function `fetch` is our own code, and is defined in terms of `reqwest`, see fetch.js line 92.